src/app/auth/duck/operations.ts

```diff
@@ -13,36 +15,69 @@ const failedTokenRequest = Creators.failedToken;
 //  jsonpath='{.clusters[*].cluster.server}')
 const loginRequest = (username, password) => {
   return dispatch => {
-    dispatch(requestTokenAction());
-    //simulate token request
+    dispatch(login(username, password));
+    return (
+      fetch(JWT_API_URL + "users", {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({
+          user: { email: username, password: password }
+        })
+        // headers: {
+        //   Authorization: "Token" + hardCodedToken
+        // },
+        // mode: "cors"
+      })
+        .then(handleResponse)
+        // .then(response => response.json())
+        .then(res => {
+          localStorage.setItem("currentUser", JSON.stringify(res.user));
+          dispatch(loginSuccess(res.user.email));
+          history.push("/");
```

Comment:
Why you use `history.push` if you can do this through redux and `connected-react-router`?